perfrunbook/debug_hw_perf.md

```diff
@@ -16,7 +16,7 @@ There are hundreds of events available to monitor in a server CPU today which is
 
 ## How to Collect PMU counters
 
-A limited subset of PMU events for the CPU are available on Graviton \*6g, \*7g sizes <16xl, we recommend using a 16xl for experiments needing PMU events to get access to all of them. On Graviton \*8g, sizes >24xl have access to all the CPU PMU events. On 5th and 6th generation x86 instances use a single socket instance is needed to have access to the CPU PMU events: >c5.9xl, >\*5.12xl, >\*6i.16xl, >c5a.12xl, and >\*6a.24xl.  On 7th generation x86 instances *7a and *7i, all sizes get access to a limited number of CPU PMU events, just like on Graviton instances, and full socket or larger instances (>\*7\*.24xl) get access to all PMU events.
```

Comment:
There's a bit of a gap here regarding 8g with sizes < 24xl.

I feel like this is getting to the point that could be done better with a table.

Reply:
Yeah, I agree, with 3 generations of each architecture its gotten hard to parse.

Reply:
Addressed with a table.  [Here](https://github.com/lrbison/aws-graviton-getting-started/tree/pmu_text/perfrunbook/debug_hw_perf.md#how-to-collect-pmu-counters) is the render direct-link.

Please scan for accuracy.

Reply:
You can display a 'rich diff' for markdown changes directly on the diff reviewing tab.